api/src/middleware/assessmentsRouter.ts

```diff
@@ -0,0 +1,45 @@
+import { Router } from 'express';
+import { itemEnvelope } from './responseEnvelope';
+
+const assessmentsRouter = Router();
+assessmentsRouter.get('/', (req, res) => {
+  const response = { behaviour: 'list of assessments' };
+  res.status(200).json(itemEnvelope(response));
+});
+
+assessmentsRouter.post('/', (req, res) => {
+    const response = { behaviour: 'create new assessment' };
+    res.status(200).json(itemEnvelope(response));
+  });
+
+assessmentsRouter.get('/:id', (req, res) => {
+    const response = { behaviour: 'view a single assessment' };
+    res.status(200).json(itemEnvelope(response));
+});
+
+assessmentsRouter.put('/:id', (req, res) => {
+    const response = { behaviour: 'Edits an assessment in the system' };
```

Comment:
I know these responses are just placeholders and they will be replaced next week, but they will form the basis for our documentation, so it'd be great if we could decide whether or not they should be capitalized. :)